src/components/main/ItemsSection.js

```diff
@@ -0,0 +1,123 @@
+    const addItemOnClickHandler = () => {window.location.href = "/additem";}
+    const searchSubmitHandler = (e) => {
+        e.preventDefault();
+        keywordSetter(e.target['search'].value);
+    }
+
+    return (
+        <div className="section-header">
+            <h3>판매 중인 상품</h3>
+            <form className="search-container" onSubmit={searchSubmitHandler}>
+                <div className="search-icon"><SearchMark /></div>
+                <input className="search-input" name="search" placeholder="검색할 상품을 입력해주세요" />
+            </form>
+            <button className="add-item-button" onClick={addItemOnClickHandler}>상품 등록하기</button>
+            <button className="sort-dropdown" onClick={dropdownHandler}>
+                {(orderBy==='recent') && <span className="dropdown-text">최신순</span>}
+                {(orderBy==='favorite') && <span className="dropdown-text">좋아요순</span>}
+                <DropdownArrow />
+            </button>
+            <div className="orderBy-dropdown hidden">
+                <button className="dropdown-button orderBy-recent" disabled={orderBy==="recent"} onClick={() => {orderSetter("recent")}}>
+                    최신순
+                </button>
+                <button className="dropdown-button orderBy-favorite" disabled={orderBy==="favorite"} onClick={() => {orderSetter("favorite")}}>
+                    좋아요순
+                </button>
+            </div>
+        </div>
+    )
+}
+
+function ItemsSectionList({items}) {
+
+    return (
+        <div className="items-list">
+            {items.map(
+                (item) => <div key={item.id} className="items-card"><Card item={item} /></div>
+            )}
+        </div>
+    )
+}
+
+function Pages({ page, pageSize, total, pageSetter }) {
+    
+    const pages = [];
+    for (let i = 1; i <= (total / pageSize) + 1; i++) pages.push(i);
+    const lastPage = pages.length;
+
+    const arrowClickHandler = (e) => {
+        if (e.target.classList.contains("prev")) {
+            pageSetter(page-1);
+        }
+        else if (e.target.classList.contains("next")) {
+            pageSetter(page+1);
+        }
+    }
+
+    return (
+        <div className="pages">
+            <button className="page-button prev" disabled={page===1} onClick={arrowClickHandler}>{'<'}</button>
+            {pages.map(
+                (i) => {return (i===page)
+                    ? <button key={i} className="page-button active" disabled={page===i} onClick={() => {pageSetter(i)}}>{i}</button>
+                    : <button key={i} className="page-button" disabled={page===i} onClick={() => {pageSetter(i)}}>{i}</button>}
+            )}
+            <button className="page-button next" disabled={page===lastPage} onClick={arrowClickHandler}>{'>'}</button>
+        </div>
+    );
+}
+
+function ItemsSection() {
+
+    const [items, setItems] = useState([]);
+    const [page, setPage] = useState(1);
+    const [pageSize, setPageSize] = useState(10);
```

Comment:
`setPageSizes`는 사용되지 않고 있어요. 이런 상태는 일반 변수로 선언을 해주시면 됩니다.

```suggestion
const pageSize = 10
```